packages/api/src/server.ts

```diff
@@ -137,11 +136,19 @@ export const createApp = (): {
   // respond healthy to auto-scaler.
   app.get('/_ah/health', (req, res) => res.sendStatus(200))
 
-  app.use('/api/auth', authRouter())
+  // 5 RPM for auth requests
+  const authLimiter = rateLimit({
+    windowMs: 60 * 1000, // 1 minute
+    max: 5,
```

Comment:
I wonder if during account creation this will be slightly too low. I'm just going to go through the flow now and see how many times its called.

Reply:
yeah, one concern is "login" could be called many times if the user is unsure about the password but probably we should also limit the number of attempts